navbar.html

```diff
@@ -42,7 +42,11 @@
         </div>
         <div br-slot="br-navbar-toolbar">
           <div class="navbar-right">
-            <div br-slot="br-navbar-tool"></div>
+            <div class="btn-toolbar">
```

Comment:
Here is the css for the btn-toolbar class: https://github.com/digitalbazaar/bedrock-angular/blob/master/css/app.css#L3329